wombat/src/main/cpp/subsystems/Arm.cpp

```diff
@@ -84,77 +95,77 @@ void Arm::OnUpdate(units::second_t dt) {
   _config.leftGearbox.transmission->SetVoltage(voltage);
   _config.rightGearbox.transmission->SetVoltage(voltage);
 
-  //creates network table instances for the angle and config of the arm
+  // creates network table instances for the angle and config of the arm
   _table->GetEntry("angle").SetDouble(angle.convert<units::degree>().value());
   _config.WriteNT(_table->GetSubTable("config"));
 }
 
-void Arm::SetArmSpeedLimit(double limit) {
-  armLimit = limit;
-}
+void wom::subsystems::Arm::SetArmSpeedLimit(double limit) { armLimit = limit; }
 
-//defines information needed for the functions and connects the states to their respective function
+// defines information needed for the functions and connects the states to their
+// respective function
 
-void Arm::SetIdle() {
-  _state = ArmState::kIdle;
-}
+void wom::subsystems::Arm::SetIdle() { _state = ArmState::kIdle; }
 
-void Arm::SetRaw(units::volt_t voltage) {
-  _state = ArmState::kRaw;
+void wom::subsystems::Arm::SetRaw(units::volt_t voltage) {
+  _state = wom::subsystems::ArmState::kRaw;
   _voltage = voltage;
 }
 
-void Arm::SetAngle(units::radian_t angle) {
-  _state = ArmState::kAngle;
+void wom::subsystems::Arm::SetAngle(units::radian_t angle) {
+  _state = wom::subsystems::ArmState::kAngle;
   _pid.SetSetpoint(angle);
 }
 
-void Arm::SetVelocity(units::radians_per_second_t velocity) {
-  _state = ArmState::kVelocity;
+void wom::subsystems::Arm::SetVelocity(units::radians_per_second_t velocity) {
+  _state = wom::subsystems::ArmState::kVelocity;
   _velocityPID.SetSetpoint(velocity);
 }
 
-ArmConfig &Arm::GetConfig() {
+wom::subsystems::ArmConfig &wom::subsystems::Arm::GetConfig() {
   return _config;
 }
 
-units::radian_t Arm::GetAngle() const {
+units::radian_t wom::subsystems::Arm::GetAngle() const {
   return _config.armEncoder.GetPosition() / 100 * 360 * 1_deg;
 }
 
-units::radians_per_second_t Arm::MaxSpeed() const {
+units::radians_per_second_t wom::subsystems::Arm::MaxSpeed() const {
   return _config.leftGearbox.motor.Speed(0_Nm, 12_V);
 }
 
-units::radians_per_second_t Arm::GetArmVelocity() const {
+units::radians_per_second_t wom::subsystems::Arm::GetArmVelocity() const {
   return _config.armEncoder.GetVelocity() / 100 * 360 * 1_deg / 60_s;
 }
 
-bool Arm::IsStable() const {
-  return _pid.IsStable(5_deg);
-}
-
+bool wom::subsystems::Arm::IsStable() const { return _pid.IsStable(5_deg); }
```

Comment:
probably want the isStable variable to be changeable

Reply:
Good idea fr fr